lib/node/NodeMainTemplate.runtime.js

```diff
@@ -18,7 +18,7 @@ module.exports = function() {
 		return Promise.resolve(update);
 	}
 
-	function hotDisposeChunk(chunkId) {
+	function hotDisposeChunk(chunkId) { //eslint-disable-line no-unused-vars
```

Comment:
Wrapped in CJS functions `module.exports` this doesn't work also with `/* exported */`